packages/node_modules/@ciscospark/widget-recents/src/enhancers/setup.js

```diff
@@ -16,15 +16,19 @@ import {
 import {updateSpaceTypeFilter, updateWidgetStatus} from '../actions';
 import {getSpaceWidgetProps} from '../selector';
 
-function getSpaceAvatar(s, props) {
-  const {sparkInstance} = props;
+export function getSpaceAvatar(s, props) {
+  const {users, sparkInstance} = props;
 
   if (!s.isDecrypting) {
     if (
       s.type === 'direct'
-      && s.toPersonId
     ) {
-      props.fetchAvatar({userId: s.toPersonId}, sparkInstance);
+      let {toPersonId} = s;
+
+      if (!toPersonId) {
```

Comment:
do you know why this is though? why do we have to use `currectUserId`? Why is `toPersonId` undefined?

Reply:
I guess I did. okay. Would it be possible to add comments on why we are doing this?

Reply:
@akoushke updated, thanks for the feedback.